openml/runs/trace.py

```diff
@@ -117,10 +117,10 @@ def get_parameters(self):
         result = {}
         # parameters have prefix 'parameter_'
         prefix = 'parameter_'
-
         for param in self.setup_string:
             key = param[len(prefix):]
-            result[key] = json.loads(self.setup_string[param])
+            value = self.setup_string[param]
+            result[key] = json.loads(value)
```

Comment:
Is there a reason you touched this file? If not, could you please revert these changes to keep this commit focused?

Reply:
Yes, I changed this to make it easier to inspect `value`. Due to reasons I forgot the json loader crashed, this issue is resolved, but it seems good to keep it like this.